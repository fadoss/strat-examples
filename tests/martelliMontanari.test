deduction/martelliMontanari

srewrite x(1) = c(1) * x(1) using marmon .
srewrite x(1) = c(1) * x(2) ; x(1) = x(3) * c(2) using marmon .
srewrite x(1) = x(2) * x(3) ; x(1) = x(3) -> x(4) using marmon .
srewrite x(1) = x(2) ; x(1) = c(1) * c(2) -> x(3) ; x(2) = x(4) -> c(3) using marmon .
