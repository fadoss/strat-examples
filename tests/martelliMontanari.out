==========================================
srewrite in MAIN : x(1) = c(1) * x(1) using marmon .

Solution 1
rewrites: 12
result [EquationSet]: fail

No more solutions.
rewrites: 12
==========================================
srewrite in MAIN : x(1) = x(3) * c(2) ; x(1) = c(1) * x(2) using marmon .

Solution 1
rewrites: 454
result NeEquationSet: x(1) = c(1) * c(2) ; x(2) = c(2) ; x(3) = c(1)

No more solutions.
rewrites: 454
==========================================
srewrite in MAIN : x(1) = x(3) -> x(4) ; x(1) = x(2) * x(3) using marmon .

Solution 1
rewrites: 94
result [EquationSet]: fail

No more solutions.
rewrites: 94
==========================================
srewrite in MAIN : x(1) = x(2) ; x(1) = c(1) * c(2) -> x(3) ; x(2) = x(4) -> c(
    3) using marmon .

Solution 1
rewrites: 2946
result NeEquationSet: x(1) = c(1) * c(2) -> c(3) ; x(2) = c(1) * c(2) -> c(3) ;
    x(3) = c(3) ; x(4) = c(1) * c(2)

No more solutions.
rewrites: 2946
Bye.
